db/schema.rb

```diff
@@ -23,7 +23,7 @@
     t.boolean "archived", default: false, null: false
     t.boolean "on_aws", default: false, null: false
     t.index ["name"], name: "index_applications_on_name", unique: true
-    t.index ["repo"], name: "index_applications_on_repo", unique: true
+    t.index ["repo"], name: "index_applications_on_repo"
```

Comment:
I don't see a corresponding migration for this change. It could be that the file is uncommitted?

Reply:
I need to read about this, do not currently know how to do this